src/gui/editors/SongEditor.cpp

```diff
@@ -546,22 +542,24 @@ void SongEditor::wheelEvent( QWheelEvent * we )
 		// and make sure, all Clip's are resized and relocated
 		realignTracks();
 	}
-
-	// FIXME: Reconsider if determining orientation is necessary in Qt6.
-	else if (std::abs(we->angleDelta().x()) > std::abs(we->angleDelta().y())) // scrolling is horizontal
-	{
-		adjustLeftRightScoll(we->angleDelta().x());
-	}
-	else if (we->modifiers() & Qt::ShiftModifier)
-	{
-		adjustLeftRightScoll(we->angleDelta().y());
-	}
 	else
 	{
+		// How much the scroll wheel moves the screen depends on zoom level
+		const float speed = 120 * DEFAULT_PIXELS_PER_BAR / pixelsPerBar();
```

Comment:
`120` is derived from line 522

    value * DEFAULT_PIXELS_PER_BAR / pixelsPerBar()

where value comes from angleDelta(), and that is 120 per wheel step